### ISSUE TITLE
Adding a Fast-Forward button smiliar to the native Rewind one

### ISSUE THREAD

**Issue by partyspy** [jwplayer/jwplayer#3894]
### Expected Behavior

A fast-forward button to seek +10 seconds next to the native rewind -10 sec one, and the button transits from the controlbar into a big button on the right part of player container when the window turns to small size, just like the rewind button.

### Actual Behavior

There is no such feature in jwplayer, orr an easy-to-go API to implement such feature. (Please correct me if im mistaken)

### Looking forward to better user experience
I think In actual use case fast-forwarding the stream is a frequent behavior. Sometime users just want to make tiny seek forwards rather than a leap one. For example, in Youtube there's this feature double-clicking the left or right side of the player to seek rewind or forward by 10 seconds. In small devices like cellphones, it's kind of tricky to scroll the progress bar to make a small forward seek, especially when the stream is long and a little scroll will be a giant leap. With iPhone there's no such difficulty because the iOS system overtakes any players with its native one, and with their native rewind/forward +15sec buttons inside. However there's no such feature in Android.

In my opinion, a easy fast-forward feature is quite important to user experience. Though I understand that adding more buttons in the control bar of player would make it too crowded, I think at least there should be such an API for users  themselves to implement it. I realize there's a `addButton` API to add buttons in control bar, but it seems there's no API to add buttons on the display container, nor a one to implement the transition.

### Environment

- Android Devices
- Chrome, Edge, etc
- *JW Player version 8.7.3*

----------------------------------------------------------------------------------------

This template is provided as a guide to filing bugs and feature requests. Please replace 
all text *in italics* with details describing your issue. When submitting a feature request, 
"Steps to reproduce" can be substituted with a user story or list of acceptance criteria.

**Comment by masterofthesith** [comment]
```
const player = jwplayer("#player");
player.on('ready', () => {
	const rewind = playerContainer.querySelector('.jw-display-icon-rewind');
	const rewDiv = $(rewind).find(".jw-icon-rewind").clone();
	$(rewDiv).addClass("forward").css("transform","scaleX(-1)").attr("aria-label","10 Saniye ileri sar").prependTo($(playerContainer).find('.jw-display-icon-next'));		
	$(playerContainer).find('.jw-display-icon-next .jw-icon-next').hide();
	const rewindS = playerContainer.querySelector('.jw-button-container');
	const $rew2Div = $(rewindS).find(".jw-icon-rewind");
	const $clone = $rew2Div.clone();
	$clone.addClass("forward").css("transform","scaleX(-1)").attr("aria-label","Forward 10 Seconds").insertAfter($rew2Div);
});
$(document).on('click','.jw-icon-rewind.forward',function(e) {
	player.seek((greplayer.getPosition() + 10));
});
```
jQuery must be imported..

**Comment by pornulus** [comment]
I've ported the snippet above to vanilla js, used a more semantic variable names and cloned the icon-rewind-container instead of only the icon itself which seemed more accurate.

```javascript
const playerContainerSelector = '#jwplayer-container';
const player = jwplayer('#player');
const playerContainer = document.querySelector(playerContainerSelector);

// display icon
const rewindContainer = playerContainer.querySelector('.jw-display-icon-rewind');
const forwardContainer = rewindContainer.cloneNode(true);
const forwardDisplayButton = forwardContainer.querySelector('.jw-icon-rewind');
forwardDisplayButton.style.transform = "scaleX(-1)";
forwardDisplayButton.ariaLabel = "Forward 10 Seconds"
const nextContainer = playerContainer.querySelector('.jw-display-icon-next');
nextContainer.parentNode.insertBefore(forwardContainer, nextContainer);


// control bar icon
playerContainer.querySelector('.jw-display-icon-next').style.display = 'none'; // hide next button
const buttonContainer = playerContainer.querySelector('.jw-button-container');
const rewindControlBarButton = buttonContainer.querySelector(".jw-icon-rewind");
const forwardControlBarButton = rewindControlBarButton.cloneNode(true);
forwardControlBarButton.style.transform = "scaleX(-1)";
forwardControlBarButton.ariaLabel = "Forward 10 Seconds";
rewindControlBarButton.parentNode.insertBefore(forwardControlBarButton, rewindControlBarButton.nextElementSibling);

// add onclick handlers
[forwardDisplayButton, forwardControlBarButton].forEach(button => {
  button.onclick = () => {
    player.seek((player.getPosition() + 10));
  }
})
```

Just be aware that the css transform obviously will mirror the 10 in the icon as well, which is not perfect - so any chance we can have this feature added to jwplayer directly?

**Comment by masterofthesith** [comment]
> I've ported the snippet above to vanilla js, used a more semantic variable names and cloned the icon-rewind-container instead of only the icon itself which seemed more accurate.
> 
> ```js
> const playerContainerSelector = '#jwplayer-container';
> const player = jwplayer('#player');
> const playerContainer = document.querySelector(playerContainerSelector);
> 
> // display icon
> const rewindContainer = playerContainer.querySelector('.jw-display-icon-rewind');
> const forwardContainer = rewindContainer.cloneNode(true);
> const forwardDisplayButton = forwardContainer.querySelector('.jw-icon-rewind');
> forwardDisplayButton.style.transform = "scaleX(-1)";
> forwardDisplayButton.ariaLabel = "Forward 10 Seconds"
> const nextContainer = playerContainer.querySelector('.jw-display-icon-next');
> nextContainer.parentNode.insertBefore(forwardContainer, nextContainer);
> 
> 
> // control bar icon
> playerContainer.querySelector('.jw-display-icon-next').style.display = 'none'; // hide next button
> const buttonContainer = playerContainer.querySelector('.jw-button-container');
> const rewindControlBarButton = buttonContainer.querySelector(".jw-icon-rewind");
> const forwardControlBarButton = rewindControlBarButton.cloneNode(true);
> forwardControlBarButton.style.transform = "scaleX(-1)";
> forwardControlBarButton.ariaLabel = "Forward 10 Seconds";
> rewindControlBarButton.parentNode.insertBefore(forwardControlBarButton, rewindControlBarButton.nextElementSibling);
> 
> // add onclick handlers
> [forwardDisplayButton, forwardControlBarButton].forEach(button => {
>   button.onclick = () => {
>     player.seek((player.getPosition() + 10));
>   }
> })
> ```
> 
> Just be aware that the css transform obviously will mirror the 10 in the icon as well, which is not perfect - so any chance we can have this feature added to jwplayer directly?

I want to help you fastly but you are right it is well but not perfect. I am using it at https://bit.ly/3BUDlLl

**Comment by pornulus** [comment]
Yeah, sure - I'm aware of the problem and I see where it comes from. Just pointing it out for anyone who comes across this :)

**Comment by mindro1232** [comment]
how should i add this on jwplayer

**Comment by TechOnlyAbdhesh** [comment]
Anyone help me let me know that how to combine this js script `
jwplayer("jwplayerDiv").setup({    
file: "content://media/external/downloads/1993",   position: 'bottom',

/* drm: {
"clearkey": {
"keyid":"",
"key":""
}
}, */

  type: "video/mp4", controls: true,
  logo: {file: "content://media/external/downloads/1668", link: "/"},
  title: "Sony Liv", abouttext: "Streamer: AHD FLIX", aboutlink: "#"

});    
`
With this js script `jwplayer().onReady(function(){
	var myRewButton = document.createElement("div");
	myRewButton.id = "myRewButton";
	myRewButton.setAttribute('style',"margin-left: 10px; margin-top: 14px; background-image: url('http://yoursite.com/rewind.png');background-repeat: no-repeat;");
	myRewButton.setAttribute('class','jw-icon jw-icon-inline jw-button-color jw-reset');
	myRewButton.setAttribute('onclick','jwplayer().seek(jwplayer().getPosition()-5)');
	var leftGroup = document.getElementsByClassName('jw-controlbar-left-group')[0];
	leftGroup.insertBefore(myRewButton, leftGroup.childNodes[0]);

	var myFFButton = document.createElement("div");
	myFFButton.id = "myFFButton";
	myFFButton.setAttribute('style',"margin-left: 10px; margin-top: 14px; background-image: url('http://yoursite.com/ff.png');background-repeat: no-repeat;");
	myFFButton.setAttribute('class','jw-icon jw-icon-inline jw-button-color jw-reset');
	myFFButton.setAttribute('onclick','jwplayer().seek(jwplayer().getPosition()+5)');
	leftGroup.insertBefore(myFFButton, leftGroup.childNodes[2]);
});`
